baseline.yaml

```diff
@@ -649,6 +649,29 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-16  
+    maturity_level: 2
+    category: Documentation
+    criteria: |
+       The project documentation MUST include the Roles
+       and Responsibilities for members of the project.
+    objective: |
+       Projects need to document the Roles and 
+       Responsibilities of the project to provide for 
```

Comment:
Should "of the project" be "of project members" or "of project participants" or something like that?

Reply:
good suggestion!  I'll change it.